polling_stations/apps/addressbase/management/commands/import_uprn_council_lookup.py

```diff
@@ -0,0 +1,31 @@
+from django.db import connection
+from django.core.management.base import BaseCommand
+from pathlib import Path
+
+
+class Command(BaseCommand):
+    """
+    Command to import a csv which maps UPRNs to Local Authority GSS codes.
+    The csv should have two columns: 'uprn' and 'lad'.
+    You probably want to generate it with 'create_uprn_council_lookup'.
+    """
+
+    def add_arguments(self, parser):
+        parser.add_argument("path", help="Path to CSV mapping UPRNs to GSS codes.")
+
+    def handle(self, *args, **kwargs):
+        self.path = Path(kwargs["path"])
+        self.table_name = "addressbase_uprntocouncil"
+
+        if not self.path.exists():
+            raise FileNotFoundError(f"No csv found at {kwargs['path']}")
+
+        cursor = connection.cursor()
+        self.stdout.write("clearing existing data..")
+        cursor.execute("TRUNCATE TABLE %s;" % (self.table_name))
+
+        self.stdout.write("importing from CSV..")
+        with self.path.open("r") as f:
+            cursor.copy_from(f, self.table_name, sep=",")
```

Comment:
If/when you come to write a unit test for this, I suspect you might find you need to write this differently. I'll stick this here just in case you need to refer to it: https://github.com/DemocracyClub/UK-Polling-Stations/commit/35371c2df37cd953cf0eaef2a9fb77e4bb512eaf